src/bp/ui-studio/src/web/reducers/logs.ts

```diff
@@ -0,0 +1,27 @@
+import { handleActions } from 'redux-actions'
+import { appendLog } from '~/actions'
+
+const defaultState: LogsReducer = {
+  logs: []
+}
+
+const reducer = handleActions(
```

Comment:
Is there a max log / do you clean up old logs ? If the studio happens to be open for a long time with a lot of logs, ui might start to be sluggish

Reply:
I've added it in another branch (limit 200 for now)